liquibase/liquibase.github.com
# 3.8.3blog

## What I Did
Added a blog for the 3.8.3 release.
## How To Test It
Blog should be readable with working links. Also, check the links on the left nav to make sure it appears as the latest news/blog item. 
One note: For some reason, I cannot add Mario as a reviewer on this PR.
